crates/ruma-macros/src/serde/incoming.rs

```diff
@@ -274,14 +283,16 @@ impl Parse for DeriveMac {
             let _: Token![!] = input.parse()?;
             let mac: Ident = input.parse()?;
 
-            if mac != "Deserialize" {
-                return Err(syn::Error::new_spanned(
+            if mac == "Debug" {
+                Ok(Self::NegativeDebug)
+            } else if mac == "Deserialize" {
+                Ok(Self::NegativeDeserialize)
+            } else {
+                Err(syn::Error::new_spanned(
                     mac,
                     "Negative incoming_derive can only be used for Deserialize",
```

Comment:
```suggestion
                    "Negative incoming_derive can only be used for Debug and Deserialize",
```

Reply:
Right. Will include this and merge on the command line.